### ISSUE TITLE
update 933160.ra

### ISSUE THREAD

**Issue by fzipi** [coreruleset/coreruleset#3021]
These must be addressed as a group:
- #2684
- #2685
- #3021
- #3022 

## Rule 933160
 ~220 words which are common in PHP code, but have a higher chance to cause false positives in natural language or other contexts.

```
Examples: 'chr', 'eval'.
```

To mitigate false positives, a regexp looks for PHP function syntax, e.g. 'eval()'.

Regexp is generated from function names in regex-assembly/933160.ra

**Comment by dune73** [comment]
Not sure what we have to do here. Doing the list update?

**Comment by fzipi** [comment]
We need to review all PHP functions and redo the classification in the 4 groups. Documenting the process, the decisions on why some words are in PLX, and maybe converge to something that can be done in an automated way.

**Comment by fzipi** [comment]
From the list of PHP functions, I'm doing a selection of words that don't have an underscore and are English words (will use @theMiddleBlue's script for this).

**Comment by M4tteoP** [comment]
Closing as completed via https://github.com/coreruleset/coreruleset/pull/3273
